libp2p/switch.nim

```diff
@@ -350,8 +351,20 @@ proc start*(s: Switch) {.public, async: (raises: [CancelledError, LPError]).} =
 
     s.peerInfo.listenAddrs.keepItIf(it notin addrs)
 
+    debug "addresses", addrs = addrs
```

Comment:
```suggestion
    debug "listen addresses", addrs = addrs
```

Reply:
i wanted to be more descriptive, what are these addresses used for? if you see log in in terminal:

`DEBUG [TIME] addresses   addresses={0x111, 0x222,  0x333}`

 what do we know here?  
 addresses for what? 
 
current log message only looks usefully if you really know what are you look at...
 
 --- 
 
if there is better log text then "listen addresses" consider using it